src/kvirc/ui/KviMenuBar.cpp

```diff
@@ -147,21 +147,6 @@ void KviMenuBar::setupHelpPopup(QMenu *pop)
 	help->addSeparator();
 	pAction = help->addAction(*(g_pIconManager->getSmallIcon(KviIconManager::Channel)),__tr2qs("Join KVIrc International Channel on freenode"));
```

Comment:
I'm not particularly wild in specifically labeling the channel as `support` since then it seems single use, such as a help channel where the only purpose is to ask questions and leave. As for the `International` part, I'll leave that up to @un1versal since he's better with words than me ^^. And I'd still like to keep `on freenode` to specify to the user that it's a trusted network and not a random 1 user hacknet in zimbabwe.